packages/websocket-plugin/src/symbols.ts

```diff
@@ -56,7 +56,7 @@ export class ConnectWebSocket {
 /**
  * Action triggered when a error ocurrs
  */
-export class WebsocketMessageError {
+export class WebSocketMessageError {
```

Comment:
This is breaking changes!

Reply:
This action doesn't work before this PR, so can't be breaking anything

Reply:
Seriously? Add test )))

Reply:
Please look at change on websocket-subject.ts line# 64, error was not handled before this PR

Reply:
Although tempting to fix this because it didn't work before we need to be careful not to make breaking changes.
Please revert this rename and we will add it to the list of breaking changes we want to introduce in v4.

Reply:
reverted